benchmark/single-source/StringTests.swift

```diff
@@ -19,6 +19,7 @@ public func run_StringWithCString(_ N: Int) {
 }
 
 public func run_StringHasPrefix(_ N: Int) {
```

Comment:
@airspeedswift Fixed in https://github.com/apple/swift/pull/7810/commits/79806234281faaadaa18dced4dc558f361aae9b1. Thanks.